common/appcommon.hpp

```diff
@@ -37,12 +37,24 @@
 #else
 extern "C" inline int sleep(int seconds) { Sleep(seconds * 1000); return 0; }
 #endif
+
+inline bool SysInitializeNetwork()
+{
+    WORD wVersionRequested = MAKEWORD(2, 2);
+    WSADATA wsaData;
+    return WSAStartup(wVersionRequested, &wsaData) == 0;
```

Comment:
This *could* be fixed by calling `srt_startup()`, but this is logically improper. This is required not for SRT (SRT does it automatically by calling `srt_startup()` together with the first socket), but for UDP, or simply any kind of system networking functions on Windows. In other words, if `stransmit` didn't use any network functions except those from SRT, this wouldn't be necessary at all.

But calling `srt_startup()` just in order to make UDP-related system function work is, you know, kinda clumsy.